utils/lean_terraform_client.py

```diff
@@ -13,3 +15,12 @@ def state_rm_access_key(working_dirs, account, user):
                  cwd=wd, stdout=PIPE, stderr=PIPE)
     proc.communicate()
     return proc.returncode == 0
+
+
+def show_json(working_dir, out_file):
+    proc = Popen(['terraform', 'show', '-json', out_file],
+                 cwd=working_dir, stdout=PIPE, stderr=PIPE)
+    out, _ = proc.communicate()
+    if proc.returncode != 0:
+        raise Exception('terraform show failed')
```

Comment:
not sure how often we will see this, but `subprocess.run()` with `check=True` gives us a very specific exception with all the debug details.

Reply:
i don't think this will ever happen.

Reply:
Famous last words :-P

Reply:
if you say "famous last words" does it cancel the jinx? :)